src/examples/colors/generate_wcs_languages.py

```diff
@@ -131,33 +139,40 @@ def get_expression_from_language(color_name:str, language:ColorLanguage):
                 language_terms[lang_num] = []
             language_terms[lang_num].append(lang_term_abbrev)
 
+    # Generate the uniform probability prior as a baseline
+    # uniform_prior = np.array(color_universe._prior)
 
-    #Generate the uniform probability prior as a baseline
-    #uniform_prior = np.array(color_universe._prior)
-
-    #Munsell to Cielab hues
+    # Munsell to Cielab hues
     munsell_to_cielab = np.array(list(munsell_to_cielab.values()))
-    #print(f"Munsell to CIELAB hues:{munsell_to_cielab}")
+    # print(f"Munsell to CIELAB hues:{munsell_to_cielab}")
 
     SIGMA_SQUARED_SCALAR = 64
-    #Calculate the meaning space as an isotropic Gaussian centered at the first chip C, for all other points
-    def meaning_distance(center, point):
-        return math.exp((-1/(2*SIGMA_SQUARED_SCALAR) * np.linalg.norm(center-point)))
 
-    #Generate the meaning space
-    meaning_dists = np.zeros(shape=(len(color_universe.referents), len(color_universe.referents)))
+    # Calculate the meaning space as an isotropic Gaussian centered at the first chip C, for all other points
+    def meaning_distance(center, point):
+        return math.exp(
+            (-1 / (2 * SIGMA_SQUARED_SCALAR) * np.linalg.norm(center - point))
+        )
+
+    # Generate the meaning space
+    meaning_dists = np.zeros(
+        shape=(len(color_universe.referents), len(color_universe.referents))
+    )
     for c1_index, c1 in enumerate(color_universe.referents):
         for c2_index, c2 in enumerate(color_universe.referents):
-            meaning_dists[c1_index][c2_index] = meaning_distance(np.array((c1.L, c1.a, c1.b)), np.array((c2.L, c2.a, c2.b)))
+            meaning_dists[c1_index][c2_index] = meaning_distance(
+                np.array((c1.L, c1.a, c1.b)), np.array((c2.L, c2.a, c2.b))
+            )
 
+    # meaning_dists = meaning_dists / meaning_dists.sum(axis=1, keepdims=True)
 
-    #meaning_dists = meaning_dists / meaning_dists.sum(axis=1, keepdims=True)
-
-    #print(language_terms)
+    # print(language_terms)
     expressions_by_speaker = {}
     language_colors_to_expressions = {}
     speakers_by_language = {}
-    with open(f"{current_dir}/data/foci-exp.txt", newline="", encoding="utf-8") as csvfile:
+    with open(
+        f"{current_dir}/data/foci-exp.txt", newline="", encoding="utf-8"
```

Comment:
argument instead of hard-coded filename

Reply:
Will make this an argument to the function